project/metals.sbt

```diff
@@ -0,0 +1,6 @@
+// DO NOT EDIT! This file is auto-generated.
```

Comment:
Is this file committed intentionally?

Reply:
good catch! Indeed I put it there by mistake, I'll remove it